src/main/java/net/fabricmc/loom/util/Mirrors.java

```diff
@@ -0,0 +1,19 @@
+package net.fabricmc.loom.util;
+
+public class Mirrors {
+	public static String LIBRARIES_BASE_MIRRORS;
+	public static String RESOURCES_BASE_MIRRORS;
+	public static String VERSION_MANIFESTS;
+
+	public static void changeMirror(String lib,String res,String manifest){
+		LIBRARIES_BASE_MIRRORS = lib;
+		RESOURCES_BASE_MIRRORS = res;
+		VERSION_MANIFESTS = manifest;
+	}
+
+	public static boolean hasMirror(){
+		return ! (LIBRARIES_BASE_MIRRORS.isEmpty()
```

Comment:
Won't this crash with an NPE since the fields are `null` (`changeMirror` not called) when none of the properties are given?

I recommend creating 3 methods `getLibrariesBase` `getResourcesBase` `getVersionManifests` than using the fields directly.